apps/web/src/server/applications/lib/services/case.service.js

```diff
@@ -93,11 +94,59 @@ export const updateCase = async (caseId, payload) => {
 };
 
 /**
- * Get documents categories for the case
+ * Get all the subfolders in a folder, or the top level folders for the case
  *
  * @param {number} id
+ * @param {number | null} folderId
  * @returns {Promise<DocumentationCategory[]>}
  */
-export const getCaseDocumentationCategories = (id) => {
-	return get(`applications/${id}/documents`);
+export const getCaseFolders = (id, folderId = null) => {
+	return folderId
+		? get(`applications/${id}/folders/${folderId}/sub-folders`)
+		: get(`applications/${id}/folders`);
+};
+
+/**
+ * Get a single folder on a case
+ *
+ * @param {number} caseId
+ * @param {number} id
+ * @returns {Promise<DocumentationCategory>}
+ */
+export const getCaseFolder = (caseId, id) => {
+	const folder = get(`applications/${caseId}/folders/${id}`);
+
+	return folder;
+};
+
+/**
```

Comment:
question: does this also return the folder itself? or just the parents?

Reply:
yes it does return the current folder too - it will be the last one in the list